src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -103,6 +110,14 @@ class SearchResultsPage extends React.Component {
                     updateIsLoadingState={this.updateIsLoadingState}
                     searchError={searchError}
                   />
+                  <span
```

Comment:
You don't actually want to render the span unless if you do have an error message, no?